image_data/office/ReadMe.md

```diff
@@ -0,0 +1,9 @@
+# Office dataset
+
+Office-31 + Caltech-10 dataset [[Original data source](http://www.stat.ucla.edu/~jxie/iFRAME/code/imageClassification.rar)]. Folder names (i.e. class names) of Caltech-10 from the original source have been renamed to be consistent with the folder names of Office-31.
```

Comment:
Caltech-10 or Caltech-101?

Reply:
Caltech-10 is the 10 classes of Caltech-256 overlapped with Office-31. It is not the official name, but used in some papers.